test/functional/tools/samples_tool_conf.xml

```diff
@@ -60,6 +60,7 @@
   <tool file="job_environment_default.xml" />
   <tool file="job_environment_default_legacy.xml" />
   <tool file="job_environment_explicit_shared_home.xml" />
+  <tool file="galaxy_slots_and_memory.xml" />
```

Comment:
@mvdbeek Did you forget to add the `galaxy_slots_and_memory.xml` file? Loading of this tool fails and the corresponding `test_slots_and_memory` integration test is skipped.